script.js

```diff
@@ -0,0 +1,228 @@
+var audioError = function(){
+  return new Howl({
+    urls: ['/home/amanda/Brackets/javaScript/simonGame/beep2.mp3'],
+    volume: 0.5,
+  })
+}
+
+var audioGreen = function(callback){
+  return  new Howl({
+    urls: ['https://s3.amazonaws.com/freecodecamp/simonSound1.mp3'],
+    volume: 0.5,
+    onend: function() {
+      $("#green").css('background','#00A74A');
+      if (callback){
+        callback();
+      }
+    }
+  });
+} 
+var audioRed = function(callback){
+  return  new Howl({
+    urls: ['https://s3.amazonaws.com/freecodecamp/simonSound2.mp3'],
+    volume: 0.5,
+    onend: function() {
+      $("#red").css('background','#9F0F17');
+      if (callback){
+        callback();
+      }
+    }
+  }); 
+}
+var audioBlue = function(callback){
+  return  new Howl({
+    urls: ['https://s3.amazonaws.com/freecodecamp/simonSound3.mp3'],
+    volume: 0.5,
+    onend: function() {
+      $("#blue").css('background','#094A8F');
+      if (callback){
+        callback();
+      }
+    }
+  }); 
+}
+var audioYellow = function(callback){
+  return  new Howl({
+    urls: ['https://s3.amazonaws.com/freecodecamp/simonSound4.mp3'],
+    volume: 0.5,
+    onend: function() {
+      $("#yellow").css('background','#CCA707');
+      if (callback){
+        callback();
+      }
+    }
+  });
+}
+
+var num;
+var aleat;
+var btnClicked;
+var typeofGame;
+var allColors=['green','red','blue','yellow'];
+var rightSequence;
+var personSequence;
+var order=0;
+
+function startGame(){
+  disableColor();
+  order=0;
+  btnClicked=true;
+  num=1;
+  rightSequence=[];
+  personSequence=[];
+  continueGame();
+}  
+
+
+function continueGame(){
+  order=0;
+  aleat=Math.floor((Math.random() * 4));
+  var choice = allColors[aleat];
+  rightSequence.push(choice);
+  disableColor();
+  doit();
+
+}
+
+
+function enableColor(){
+  $("span").css("pointer-events", "auto");
+}
+
+function disableColor(){
+  $("span").css("pointer-events", "none");
+}
+
+
+function doit(){
+  $("#number").html(num);
+  var counter=0; //define um contador para cada chamada;
+  for (var i in rightSequence){
+    var color=rightSequence[i];
+    (function(color){   // encapsulo o setTimeOut para nao executar de uma vez
+      setTimeout(function(){
+        lightSound(color);       
+      },800 * ++counter);
+
+    })(color);
+  }
+  (function(){
+    setTimeout(function(){
+      enableColor();
+    }, 800 * counter+1);
+  })();
+}
+
+function lightSound(cor, callback){
+  switch (cor){
```

Comment:
You are duplicating code around all the cases, create a function like in the examples before.